ripe/atlas/cousteau/request.py

```diff
@@ -24,17 +24,14 @@ class AtlasRequest(object):
     }
 
     def __init__(self, **kwargs):
+
         self.url = ""
-        self.url_path = ""
-        self.key = kwargs.get("key", None)
-        if "url_path" in kwargs:
-            self.url_path = kwargs["url_path"]
-        if "server" in kwargs:
-            self.server = kwargs["server"]
-        else:
-            self.server = "atlas.ripe.net"
+        self.key = kwargs.get("key")
```

Comment:
does this mean that the key is guaranteed to be there now? (as you dropped the None part...)